mlfinlab/codependence/codependence_matrix.py

```diff
@@ -70,12 +75,14 @@ def get_dependence_matrix(df: pd.DataFrame, dependence_method: str) -> pd.DataFr
 
 def get_distance_matrix(X: pd.DataFrame, distance_metric: str = 'angular') -> pd.DataFrame:
     """
-    Apply distance operator to a dependence matrix.
+    Applies distance operator to a dependence matrix.
+
+    This allows to turn a correlation matrix into a distance matrix. Distances used are true metrics.
 
     :param X: (pd.DataFrame) Dataframe to which distance operator to be applied.
     :param distance_metric: (str) The distance operator to be used for generating the distance matrix.
-       The methods that can be applied are: 'angular', 'squared_angular' and 'absolute_angular'.
-    :return: (pd.DataFrame) Distance matrix
+                                  Supported methods are: 'angular', 'squared_angular' and 'absolute_angular'.
```

Comment:
You can change these to - Supported methods are: ``angular``, ``squared_angular`` etc... This just shows up more nicely on read-the-docs. I have done the same for portfolio optimisation docs. You can give it a look to see how it is rendered

Reply:
Thanks. I'll make this change. 🙂